src/backend/aspen/database/models/usergroup.py

```diff
@@ -20,6 +20,11 @@ class Group(idbase, DictMixin):  # type: ignore
 
     name = Column(String, unique=True, nullable=False)
     address = Column(String, nullable=True)
+    prefix = Column(
+        String,
+        nullable=True,
```

Comment:
Do we want this to be nullable?

Reply:
Probably also want to enforce it as unique

Reply:
yeah uniqueness sounds good! I left it as nullable in case we had a group that was just for viewing (like how we have the admin group, i imagine we could also have some view only groups?), do you think that assesment is right?

Reply:
i changed it to be not null